airflow/providers/databricks/hooks/databricks_sql.py

```diff
@@ -35,7 +37,12 @@
 
 from databricks import sql  # type: ignore[attr-defined]
 
-from airflow.exceptions import AirflowException, AirflowProviderDeprecationWarning
+from airflow.exceptions import (
+    AirflowException,
+    AirflowProviderDeprecationWarning,
+    AirflowTaskExecutionError,
+    AirflowTaskExecutionTimeout,
```

Comment:
You can’t do this because providers have backward compatibility obligations and need to work on older Airflow versions. Since these are only used in this provider anyway, the exceptions can simply be declared in the provider instead.